tests/commons/j2/test_j2_functions.py

```
@@ -11,290 +12,324 @@
+            "floats disguised as strings should not be accepted as int value"
+        )
+
+
+@pytest.mark.parametrize(
+    "case",
+    [
+        ((1.0, "'1.0'^^xsd:double")),
+        (("1", "'1.0'^^xsd:double")),
+        ((1, "'1.0'^^xsd:double")),
+        ((float(1), "'1.0'^^xsd:double")),
+        ((1.00, "'1.0'^^xsd:double")),
```

Ruff (0.8.2)

Duplicate of test case at index 0 in `pytest.mark.parametrize`

Remove duplicate test case

(PT014)